src/storage/txn/actions/acquire_pessimistic_lock.rs

```diff
@@ -432,6 +451,29 @@ pub mod tests {
             true,
             false,
             TimeStamp::zero(),
+            lock_only_if_exists,
+        )
+    }
+
+    #[cfg(test)]
+    pub fn must_err_lock_only_if_exists<E: Engine>(
```

Comment:
I think it's unnecessary to create this function. Its only use is to check that `lock_only_if_exists` must be used together with `need_value`. So this is not something like a util function that is often reused.

Reply:
done